.github/workflows/_legacy-checkpoints.yml

```diff
@@ -108,6 +108,7 @@ jobs:
         retention-days: ${{ env.KEEP_DAYS }}
 
     - name: Install AWS CLI
+      continue-on-error: true
```

Comment:
Why not `which aws` and install if it's not available?

Reply:
It makes it just more complex...